applications/mobu/values-idfint.yaml

```diff
@@ -104,5 +104,6 @@ config:
       business:
         type: "SIAQuerySetRunner"
         options:
+          sync: false
```

Comment:
Do we also want to do this for the `tap` business?

Reply:
Actually I pasted that at the wrong place, it should be in the TAP business not SIA